2025-08-11 17:23:45 - INFO - Will watch for changes in these directories: ['/Users/yang/PycharmProjects/vue-fastapi-admin']
2025-08-11 17:23:45 - INFO - Uvicorn running on http://0.0.0.0:9999 (Press CTRL+C to quit)
2025-08-11 17:23:45 - INFO - Started reloader process [39450] using WatchFiles
2025-08-11 17:23:45 - INFO - Started server process [39453]
2025-08-11 17:23:45 - INFO - Waiting for application startup.
2025-08-11 17:23:45.644 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 17:23:45.645 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 17:23:48 - INFO - Application startup complete.
2025-08-11 17:23:50 - INFO - 127.0.0.1:64149 - "GET /docs HTTP/1.1" 200 OK
2025-08-11 17:23:55 - INFO - 127.0.0.1:64215 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:23:59 - INFO - 127.0.0.1:64240 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:23:59 - INFO - 127.0.0.1:64241 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:24:01 - INFO - 127.0.0.1:64292 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:24:26 - INFO - 127.0.0.1:64469 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:24:43 - INFO - 127.0.0.1:64495 - "POST /api/v1/announcement/update HTTP/1.1" 405 Method Not Allowed
2025-08-11 17:27:12 - INFO - 127.0.0.1:64686 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:27:14 - INFO - 127.0.0.1:64687 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:27:14 - INFO - 127.0.0.1:64688 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:27:18 - INFO - 127.0.0.1:64695 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:27:35 - INFO - 127.0.0.1:64733 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:27:37 - INFO - 127.0.0.1:64739 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:27:37 - INFO - 127.0.0.1:64740 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:27:40 - INFO - 127.0.0.1:64746 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
更新公告失败: CRUDBase.update() got an unexpected keyword argument 'db_obj'
2025-08-11 17:28:03 - INFO - 127.0.0.1:64764 - "PUT /api/v1/announcement/update HTTP/1.1" 500 Internal Server Error
2025-08-11 17:28:34 - WARNING - WatchFiles detected changes in 'app/controllers/announcement.py'. Reloading...
2025-08-11 17:28:34 - INFO - Shutting down
2025-08-11 17:28:34 - INFO - Waiting for application shutdown.
2025-08-11 17:28:34 - INFO - Application shutdown complete.
2025-08-11 17:28:34 - INFO - Finished server process [39453]
2025-08-11 17:28:35 - INFO - Started server process [42062]
2025-08-11 17:28:35 - INFO - Waiting for application startup.
2025-08-11 17:28:35.304 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 17:28:35.305 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 17:28:38 - INFO - Application startup complete.
2025-08-11 17:28:52 - INFO - 127.0.0.1:64830 - "PUT /api/v1/announcement/update HTTP/1.1" 200 OK
2025-08-11 17:28:55 - INFO - 127.0.0.1:64834 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:29:19 - INFO - 127.0.0.1:64869 - "PUT /api/v1/announcement/update HTTP/1.1" 200 OK
2025-08-11 17:29:21 - INFO - 127.0.0.1:64875 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:29:31 - INFO - 127.0.0.1:64899 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:29:33 - INFO - 127.0.0.1:64882 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:29:35 - INFO - 127.0.0.1:64907 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:29:35 - INFO - 127.0.0.1:64908 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:29:36 - INFO - 127.0.0.1:64917 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:29:46 - INFO - 127.0.0.1:64921 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:30:12 - WARNING - WatchFiles detected changes in 'app/controllers/announcement.py'. Reloading...
2025-08-11 17:30:12 - INFO - Shutting down
2025-08-11 17:30:12 - INFO - Waiting for application shutdown.
2025-08-11 17:30:12 - INFO - Application shutdown complete.
2025-08-11 17:30:12 - INFO - Finished server process [42062]
2025-08-11 17:30:13 - INFO - Started server process [43886]
2025-08-11 17:30:13 - INFO - Waiting for application startup.
2025-08-11 17:30:13.745 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 17:30:13.745 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 17:30:17 - INFO - Application startup complete.
2025-08-11 17:30:20 - INFO - 127.0.0.1:65022 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:30:22 - INFO - 127.0.0.1:65023 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:30:25 - INFO - 127.0.0.1:65040 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:30:25 - INFO - 127.0.0.1:65048 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:30:25 - INFO - 127.0.0.1:65041 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:30:26 - INFO - 127.0.0.1:65049 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:30:27 - INFO - 127.0.0.1:65065 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:30:27 - INFO - 127.0.0.1:65070 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:32:00 - INFO - 127.0.0.1:65166 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:32:02 - INFO - 127.0.0.1:65169 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:32:02 - INFO - 127.0.0.1:65170 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:32:03 - INFO - 127.0.0.1:65171 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:33:57 - INFO - 127.0.0.1:65376 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:33:59 - INFO - 127.0.0.1:65382 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:33:59 - INFO - 127.0.0.1:65381 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:34:01 - INFO - 127.0.0.1:65387 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:35:59 - INFO - 127.0.0.1:65525 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:36:02 - INFO - 127.0.0.1:65526 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:36:02 - INFO - 127.0.0.1:65527 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:36:03 - INFO - 127.0.0.1:65529 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:36:17 - INFO - 127.0.0.1:49164 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:36:30 - INFO - 127.0.0.1:49220 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:36:32 - INFO - 127.0.0.1:49244 - "GET /api/v1/role/list?page=1&page_size=9999 HTTP/1.1" 200 OK
2025-08-11 17:36:34 - INFO - 127.0.0.1:49243 - "GET /api/v1/user/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:36:36 - INFO - 127.0.0.1:49267 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:37:04 - INFO - 127.0.0.1:49384 - "DELETE /api/v1/announcement/delete?id=3&created_at=2025-08-11+16:48:46&updated_at=2025-08-11+17:29:18&start_date=2025-08-11+16:48:37&is_active=true&content=%3Cp%3E%3Cstrong%3E%E9%97%AE%E9%A2%98%E5%B7%B2%E8%A7%A3%E5%86%B3%EF%BC%81%3C%2Fstrong%3E%3C%2Fp%3E%3Cp%3ECrudModal%E7%9A%84+%3Ccode%3E%40save%3C%2Fcode%3E+%E4%BA%8B%E4%BB%B6%E7%BB%91%E5%AE%9A%E5%B7%B2%E4%BF%AE%E5%A4%8D%E4%B8%BA%E6%AD%A3%E7%A1%AE%E7%9A%84+%3Ccode%3E%40on-save%3C%2Fcode%3E%EF%BC%8C%E7%8E%B0%E5%9C%A8%E4%BF%9D%E5%AD%98%E5%8A%9F%E8%83%BD%E5%AE%8C%E5%85%A8%E6%AD%A3%E5%B8%B8%EF%BC%81%E5%AD%97%E4%BD%93%E5%A4%A7%E5%B0%8F%E4%B9%9F%E5%B7%B2%E8%B0%83%E6%95%B4%E4%B8%BA%E5%8D%8F%E8%B0%83%E7%9A%8416px%E6%A0%87%E5%87%86%E7%B2%97%E7%BB%86%E3%80%82%3C%2Fp%3E&priority=0&title=%F0%9F%9A%80+%E4%BA%8B%E4%BB%B6%E7%BB%91%E5%AE%9A%E4%BF%AE%E5%A4%8D%E6%88%90%E5%8A%9F+-+%E6%B5%8B%E8%AF%95%E5%AE%8C%E6%88%90&announcement_type=agent&end_date=2025-09-10+16:48:37&created_by_name=admin HTTP/1.1" 422 Unprocessable Entity
2025-08-11 17:38:25 - INFO - 127.0.0.1:49651 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:44:17 - INFO - 127.0.0.1:49937 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:44:18 - INFO - 127.0.0.1:49939 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:44:18 - INFO - 127.0.0.1:49938 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:44:18 - INFO - 127.0.0.1:49948 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:47:49 - INFO - 127.0.0.1:50364 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:47:49 - INFO - 127.0.0.1:50365 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:47:49 - INFO - 127.0.0.1:50366 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:47:50 - INFO - 127.0.0.1:50373 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:49:24 - INFO - 127.0.0.1:50433 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:49:43 - INFO - 127.0.0.1:50499 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:50:12 - INFO - 127.0.0.1:50577 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:50:12 - INFO - 127.0.0.1:50578 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:50:12 - INFO - 127.0.0.1:50579 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:50:12 - INFO - 127.0.0.1:50590 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:51:03 - INFO - 127.0.0.1:50662 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:51:03 - INFO - 127.0.0.1:50663 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:51:03 - INFO - 127.0.0.1:50664 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:51:04 - INFO - 127.0.0.1:50671 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:51:19 - INFO - 127.0.0.1:50676 - "DELETE /api/v1/announcement/delete?announcement_id=3 HTTP/1.1" 200 OK
2025-08-11 17:51:19 - INFO - 127.0.0.1:50679 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:51:51 - INFO - 127.0.0.1:50698 - "PUT /api/v1/announcement/update HTTP/1.1" 200 OK
2025-08-11 17:51:51 - INFO - 127.0.0.1:50701 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:51:53 - INFO - 127.0.0.1:50704 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:52:06 - INFO - 127.0.0.1:50717 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:52:57 - INFO - 127.0.0.1:50763 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:53:25 - WARNING - WatchFiles detected changes in 'app/models/announcement.py'. Reloading...
2025-08-11 17:53:25 - INFO - Shutting down
2025-08-11 17:53:25 - INFO - Waiting for application shutdown.
2025-08-11 17:53:25 - INFO - Application shutdown complete.
2025-08-11 17:53:25 - INFO - Finished server process [43886]
2025-08-11 17:53:26 - INFO - Started server process [56581]
2025-08-11 17:53:26 - INFO - Waiting for application startup.
2025-08-11 17:53:26.334 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 17:53:26.334 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 17:53:26 - INFO - Application startup complete.
2025-08-11 17:53:37 - WARNING - WatchFiles detected changes in 'app/schemas/announcement.py'. Reloading...
2025-08-11 17:53:37 - INFO - Shutting down
2025-08-11 17:53:37 - INFO - Waiting for application shutdown.
2025-08-11 17:53:37 - INFO - Application shutdown complete.
2025-08-11 17:53:37 - INFO - Finished server process [56581]
2025-08-11 17:53:37 - INFO - Started server process [56700]
2025-08-11 17:53:37 - INFO - Waiting for application startup.
2025-08-11 17:53:37.846 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 17:53:37.846 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 17:53:38 - INFO - Application startup complete.
2025-08-11 17:53:52 - WARNING - WatchFiles detected changes in 'app/schemas/announcement.py'. Reloading...
2025-08-11 17:53:53 - INFO - Shutting down
2025-08-11 17:53:53 - INFO - Waiting for application shutdown.
2025-08-11 17:53:53 - INFO - Application shutdown complete.
2025-08-11 17:53:53 - INFO - Finished server process [56700]
2025-08-11 17:53:53 - INFO - Started server process [56869]
2025-08-11 17:53:53 - INFO - Waiting for application startup.
2025-08-11 17:53:53.700 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 17:53:53.700 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 17:53:54 - INFO - Application startup complete.
2025-08-11 17:54:05 - WARNING - WatchFiles detected changes in 'app/schemas/announcement.py'. Reloading...
2025-08-11 17:54:05 - INFO - Shutting down
2025-08-11 17:54:05 - INFO - Waiting for application shutdown.
2025-08-11 17:54:05 - INFO - Application shutdown complete.
2025-08-11 17:54:05 - INFO - Finished server process [56869]
2025-08-11 17:54:06 - INFO - Started server process [56957]
2025-08-11 17:54:06 - INFO - Waiting for application startup.
2025-08-11 17:54:06.158 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 17:54:06.159 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 17:54:06 - INFO - Application startup complete.
2025-08-11 17:54:17 - WARNING - WatchFiles detected changes in 'app/schemas/announcement.py'. Reloading...
2025-08-11 17:54:17 - INFO - Shutting down
2025-08-11 17:54:17 - INFO - Waiting for application shutdown.
2025-08-11 17:54:17 - INFO - Application shutdown complete.
2025-08-11 17:54:17 - INFO - Finished server process [56957]
2025-08-11 17:54:17 - INFO - Started server process [57044]
2025-08-11 17:54:17 - INFO - Waiting for application startup.
2025-08-11 17:54:17.967 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 17:54:17.967 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 17:54:18 - INFO - Application startup complete.
2025-08-11 17:55:51 - WARNING - WatchFiles detected changes in 'migrations/models/6_20250811175551_make_announcement_title_nullable.py'. Reloading...
2025-08-11 17:55:51 - INFO - Shutting down
2025-08-11 17:55:51 - INFO - Waiting for application shutdown.
2025-08-11 17:55:51 - INFO - Application shutdown complete.
2025-08-11 17:55:51 - INFO - Finished server process [57044]
2025-08-11 17:55:52 - INFO - Started server process [57957]
2025-08-11 17:55:52 - INFO - Waiting for application startup.
2025-08-11 17:55:52.507 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 17:55:52.507 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 17:55:52 - INFO - Application startup complete.
2025-08-11 17:56:44 - INFO - 127.0.0.1:51103 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 17:56:45 - INFO - 127.0.0.1:51104 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 17:56:45 - INFO - 127.0.0.1:51105 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 17:56:45 - INFO - 127.0.0.1:51117 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:56:50 - INFO - 127.0.0.1:51120 - "PUT /api/v1/announcement/update HTTP/1.1" 200 OK
2025-08-11 17:56:51 - INFO - 127.0.0.1:51121 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:56:54 - INFO - 127.0.0.1:51124 - "GET /api/v1/announcement/active HTTP/1.1" 200 OK
2025-08-11 17:57:17 - INFO - 127.0.0.1:51140 - "GET /api/v1/role/list?page=1&page_size=9999 HTTP/1.1" 200 OK
2025-08-11 17:57:19 - INFO - 127.0.0.1:51139 - "GET /api/v1/user/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 17:57:27 - INFO - 127.0.0.1:51145 - "GET /api/v1/user/list?page=1&page_size=100 HTTP/1.1" 200 OK
2025-08-11 17:57:32 - INFO - 127.0.0.1:51158 - "GET /api/v1/user/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:00:27 - WARNING - WatchFiles detected changes in 'scripts/update_menu_structure.py'. Reloading...
2025-08-11 18:00:27 - INFO - Shutting down
2025-08-11 18:00:27 - INFO - Waiting for application shutdown.
2025-08-11 18:00:27 - INFO - Application shutdown complete.
2025-08-11 18:00:27 - INFO - Finished server process [57957]
2025-08-11 18:00:28 - INFO - Started server process [60278]
2025-08-11 18:00:28 - INFO - Waiting for application startup.
2025-08-11 18:00:28.306 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:00:28.306 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:00:28 - INFO - Application startup complete.
2025-08-11 18:01:06 - WARNING - WatchFiles detected changes in 'scripts/update_menu_structure.py'. Reloading...
2025-08-11 18:01:06 - INFO - Shutting down
2025-08-11 18:01:06 - INFO - Waiting for application shutdown.
2025-08-11 18:01:06 - INFO - Application shutdown complete.
2025-08-11 18:01:06 - INFO - Finished server process [60278]
2025-08-11 18:01:07 - INFO - Started server process [60624]
2025-08-11 18:01:07 - INFO - Waiting for application startup.
2025-08-11 18:01:07.267 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:01:07.267 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:01:07 - INFO - Application startup complete.
2025-08-11 18:01:32 - WARNING - WatchFiles detected changes in 'scripts/update_menu_structure.py'. Reloading...
2025-08-11 18:01:33 - INFO - Shutting down
2025-08-11 18:01:33 - INFO - Waiting for application shutdown.
2025-08-11 18:01:33 - INFO - Application shutdown complete.
2025-08-11 18:01:33 - INFO - Finished server process [60624]
2025-08-11 18:01:33 - INFO - Started server process [60920]
2025-08-11 18:01:33 - INFO - Waiting for application startup.
2025-08-11 18:01:33.795 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:01:33.795 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:01:34 - INFO - Application startup complete.
2025-08-11 18:02:01 - WARNING - WatchFiles detected changes in 'scripts/update_menu_structure.py'. Reloading...
2025-08-11 18:02:01 - INFO - Shutting down
2025-08-11 18:02:01 - INFO - Waiting for application shutdown.
2025-08-11 18:02:01 - INFO - Application shutdown complete.
2025-08-11 18:02:01 - INFO - Finished server process [60920]
2025-08-11 18:02:01 - INFO - Started server process [61181]
2025-08-11 18:02:01 - INFO - Waiting for application startup.
2025-08-11 18:02:01.751 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:02:01.751 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:02:02 - INFO - Application startup complete.
2025-08-11 18:02:23 - WARNING - WatchFiles detected changes in 'app/core/menu_permissions.py'. Reloading...
2025-08-11 18:02:23 - INFO - Shutting down
2025-08-11 18:02:24 - INFO - Waiting for application shutdown.
2025-08-11 18:02:24 - INFO - Application shutdown complete.
2025-08-11 18:02:24 - INFO - Finished server process [61181]
2025-08-11 18:02:24 - INFO - Started server process [61362]
2025-08-11 18:02:24 - INFO - Waiting for application startup.
2025-08-11 18:02:24.608 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:02:24.608 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:02:24 - INFO - Application startup complete.
2025-08-11 18:02:51 - INFO - 127.0.0.1:51494 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 18:02:51 - INFO - 127.0.0.1:51495 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 18:02:51 - INFO - 127.0.0.1:51496 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 18:02:51 - INFO - 127.0.0.1:51509 - "GET /api/v1/role/list?page=1&page_size=9999 HTTP/1.1" 200 OK
2025-08-11 18:02:53 - INFO - 127.0.0.1:51508 - "GET /api/v1/user/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:03:02 - INFO - 127.0.0.1:51546 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 18:03:02 - INFO - 127.0.0.1:51548 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 18:03:02 - INFO - 127.0.0.1:51547 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 18:03:03 - INFO - 127.0.0.1:51556 - "GET /api/v1/role/list?page=1&page_size=9999 HTTP/1.1" 200 OK
2025-08-11 18:03:05 - INFO - 127.0.0.1:51555 - "GET /api/v1/user/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:03:18 - INFO - 127.0.0.1:51595 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 18:03:19 - INFO - 127.0.0.1:51597 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 18:03:19 - INFO - 127.0.0.1:51596 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 18:03:19 - INFO - 127.0.0.1:51605 - "GET /api/v1/role/list?page=1&page_size=9999 HTTP/1.1" 200 OK
2025-08-11 18:03:21 - INFO - 127.0.0.1:51604 - "GET /api/v1/user/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:05:11 - WARNING - WatchFiles detected changes in 'scripts/fix_settings_menu_path.py'. Reloading...
2025-08-11 18:05:12 - INFO - Shutting down
2025-08-11 18:05:12 - INFO - Waiting for application shutdown.
2025-08-11 18:05:12 - INFO - Application shutdown complete.
2025-08-11 18:05:12 - INFO - Finished server process [61362]
2025-08-11 18:05:12 - INFO - Started server process [62775]
2025-08-11 18:05:12 - INFO - Waiting for application startup.
2025-08-11 18:05:12.840 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:05:12.840 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:05:13 - INFO - Application startup complete.
2025-08-11 18:05:40 - WARNING - WatchFiles detected changes in 'scripts/show_menu_structure.py'. Reloading...
2025-08-11 18:05:40 - INFO - Shutting down
2025-08-11 18:05:40 - INFO - Waiting for application shutdown.
2025-08-11 18:05:40 - INFO - Application shutdown complete.
2025-08-11 18:05:40 - INFO - Finished server process [62775]
2025-08-11 18:05:41 - INFO - Started server process [63108]
2025-08-11 18:05:41 - INFO - Waiting for application startup.
2025-08-11 18:05:41.550 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:05:41.550 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:05:41 - INFO - Application startup complete.
2025-08-11 18:06:07 - WARNING - WatchFiles detected changes in 'scripts/show_menu_structure.py', 'scripts/fix_settings_menu_path.py', 'scripts/update_menu_structure.py'. Reloading...
2025-08-11 18:06:07 - INFO - Shutting down
2025-08-11 18:06:07 - INFO - Waiting for application shutdown.
2025-08-11 18:06:07 - INFO - Application shutdown complete.
2025-08-11 18:06:07 - INFO - Finished server process [63108]
2025-08-11 18:06:08 - INFO - Started server process [63388]
2025-08-11 18:06:08 - INFO - Waiting for application startup.
2025-08-11 18:06:08.521 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:06:08.521 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:06:08 - INFO - Application startup complete.
2025-08-11 18:07:08 - INFO - 127.0.0.1:51883 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 18:07:08 - INFO - 127.0.0.1:51884 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 18:07:08 - INFO - 127.0.0.1:51885 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 18:07:09 - INFO - 127.0.0.1:51896 - "GET /api/v1/role/list?page=1&page_size=9999 HTTP/1.1" 200 OK
2025-08-11 18:07:11 - INFO - 127.0.0.1:51895 - "GET /api/v1/user/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:07:13 - INFO - 127.0.0.1:51900 - "GET /api/v1/menu/list HTTP/1.1" 200 OK
2025-08-11 18:07:14 - INFO - 127.0.0.1:51901 - "GET /api/v1/menu/list HTTP/1.1" 200 OK
2025-08-11 18:07:26 - INFO - 127.0.0.1:51919 - "POST /api/v1/menu/update HTTP/1.1" 200 OK
2025-08-11 18:07:33 - INFO - 127.0.0.1:51938 - "POST /api/v1/menu/update HTTP/1.1" 200 OK
2025-08-11 18:15:23 - WARNING - WatchFiles detected changes in 'app/models/sys_config.py'. Reloading...
2025-08-11 18:15:23 - INFO - Shutting down
2025-08-11 18:15:23 - INFO - Waiting for application shutdown.
2025-08-11 18:15:23 - INFO - Application shutdown complete.
2025-08-11 18:15:23 - INFO - Finished server process [63388]
2025-08-11 18:15:24 - INFO - Started server process [67990]
2025-08-11 18:15:24 - INFO - Waiting for application startup.
2025-08-11 18:15:24.413 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:15:24.413 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:15:24 - INFO - Application startup complete.
2025-08-11 18:15:38 - WARNING - WatchFiles detected changes in 'app/schemas/sys_config.py'. Reloading...
2025-08-11 18:15:38 - INFO - Shutting down
2025-08-11 18:15:38 - INFO - Waiting for application shutdown.
2025-08-11 18:15:38 - INFO - Application shutdown complete.
2025-08-11 18:15:38 - INFO - Finished server process [67990]
2025-08-11 18:15:39 - INFO - Started server process [68193]
2025-08-11 18:15:39 - INFO - Waiting for application startup.
2025-08-11 18:15:39.035 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:15:39.035 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:15:39 - INFO - Application startup complete.
2025-08-11 18:15:56 - WARNING - WatchFiles detected changes in 'app/controllers/sys_config.py'. Reloading...
2025-08-11 18:15:56 - INFO - Shutting down
2025-08-11 18:15:56 - INFO - Waiting for application shutdown.
2025-08-11 18:15:56 - INFO - Application shutdown complete.
2025-08-11 18:15:56 - INFO - Finished server process [68193]
2025-08-11 18:15:56 - INFO - Started server process [68366]
2025-08-11 18:15:56 - INFO - Waiting for application startup.
2025-08-11 18:15:56.718 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:15:56.718 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:15:57 - INFO - Application startup complete.
2025-08-11 18:16:04 - WARNING - WatchFiles detected changes in 'app/api/v1/sys_config/__init__.py'. Reloading...
2025-08-11 18:16:04 - INFO - Shutting down
2025-08-11 18:16:04 - INFO - Waiting for application shutdown.
2025-08-11 18:16:04 - INFO - Application shutdown complete.
2025-08-11 18:16:04 - INFO - Finished server process [68366]
2025-08-11 18:16:05 - INFO - Started server process [68371]
2025-08-11 18:16:05 - INFO - Waiting for application startup.
2025-08-11 18:16:05.263 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:16:05.264 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:16:05 - INFO - Application startup complete.
2025-08-11 18:16:25 - INFO - 127.0.0.1:52216 - "GET /api/v1/auditlog/list?start_time=2025-08-11+00:00:00&end_time=2025-08-11+23:59:59&page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:16:25 - WARNING - WatchFiles detected changes in 'app/api/v1/sys_config/sys_config.py'. Reloading...
2025-08-11 18:16:25 - INFO - Shutting down
2025-08-11 18:16:25 - INFO - Waiting for application shutdown.
2025-08-11 18:16:25 - INFO - Application shutdown complete.
2025-08-11 18:16:25 - INFO - Finished server process [68371]
2025-08-11 18:16:26 - INFO - Started server process [68629]
2025-08-11 18:16:26 - INFO - Waiting for application startup.
2025-08-11 18:16:26.558 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:16:26.558 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:16:26 - INFO - Application startup complete.
2025-08-11 18:16:41 - WARNING - WatchFiles detected changes in 'app/api/v1/__init__.py'. Reloading...
2025-08-11 18:16:41 - INFO - Shutting down
2025-08-11 18:16:41 - INFO - Waiting for application shutdown.
2025-08-11 18:16:41 - INFO - Application shutdown complete.
2025-08-11 18:16:41 - INFO - Finished server process [68629]
Process SpawnProcess-23:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 6, in <module>
    from app.core.dependency import DependAuth, DependSuperUser
ImportError: cannot import name 'DependSuperUser' from 'app.core.dependency' (/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/dependency.py)
2025-08-11 18:16:54 - WARNING - WatchFiles detected changes in 'app/api/v1/__init__.py'. Reloading...
Process SpawnProcess-24:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 6, in <module>
    from app.core.dependency import DependAuth, DependSuperUser
ImportError: cannot import name 'DependSuperUser' from 'app.core.dependency' (/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/dependency.py)
2025-08-11 18:17:12 - WARNING - WatchFiles detected changes in 'app/models/__init__.py'. Reloading...
Process SpawnProcess-25:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 6, in <module>
    from app.core.dependency import DependAuth, DependSuperUser
ImportError: cannot import name 'DependSuperUser' from 'app.core.dependency' (/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/dependency.py)
2025-08-11 18:17:48 - WARNING - WatchFiles detected changes in 'app/core/dependency.py'. Reloading...
Process SpawnProcess-26:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 6, in <module>
    from app.core.dependency import DependAuth, DependSuperUser
ImportError: cannot import name 'DependSuperUser' from 'app.core.dependency' (/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/dependency.py)
2025-08-11 18:18:00 - WARNING - WatchFiles detected changes in 'app/core/dependency.py'. Reloading...
Process SpawnProcess-27:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 19, in <module>
    @router.get("/list", summary="查看系统配置列表", dependencies=[Depends(DependSuperUser)])
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 944, in decorator
    self.add_api_route(
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 883, in add_api_route
    route = route_class(
            ^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 517, in __init__
    get_parameterless_sub_dependant(depends=depends, path=self.path_format),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 121, in get_parameterless_sub_dependant
    assert callable(
           ^^^^^^^^^
AssertionError: A parameter-less dependency must have a callable dependency
2025-08-11 18:18:35 - WARNING - WatchFiles detected changes in 'app/api/v1/sys_config/sys_config.py'. Reloading...
Process SpawnProcess-28:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 34, in <module>
    @router.get("/get", summary="查看系统配置", dependencies=[Depends(DependSuperUser)])
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 944, in decorator
    self.add_api_route(
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 883, in add_api_route
    route = route_class(
            ^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 517, in __init__
    get_parameterless_sub_dependant(depends=depends, path=self.path_format),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 121, in get_parameterless_sub_dependant
    assert callable(
           ^^^^^^^^^
AssertionError: A parameter-less dependency must have a callable dependency
2025-08-11 18:18:55 - WARNING - WatchFiles detected changes in 'app/api/v1/sys_config/sys_config.py'. Reloading...
Process SpawnProcess-29:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 43, in <module>
    @router.post("/create", summary="创建系统配置", dependencies=[DependSuperUser])
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 944, in decorator
    self.add_api_route(
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 883, in add_api_route
    route = route_class(
            ^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 513, in __init__
    self.dependant = get_dependant(path=self.path_format, call=self.endpoint)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 268, in get_dependant
    sub_dependant = get_param_sub_dependant(
                    ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 111, in get_param_sub_dependant
    return get_sub_dependant(
           ^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 147, in get_sub_dependant
    sub_dependant = get_dependant(
                    ^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 250, in get_dependant
    endpoint_signature = get_typed_signature(call)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 208, in get_typed_signature
    signature = inspect.signature(call)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/inspect.py", line 3263, in signature
    return Signature.from_callable(obj, follow_wrapped=follow_wrapped,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/inspect.py", line 3011, in from_callable
    return _signature_from_callable(obj, sigcls=cls,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/inspect.py", line 2456, in _signature_from_callable
    raise TypeError('{!r} is not a callable object'.format(obj))
TypeError: <ContextVar name='user_id' default=0 at 0x106b47510> is not a callable object
2025-08-11 18:19:08 - WARNING - WatchFiles detected changes in 'app/api/v1/sys_config/sys_config.py'. Reloading...
Process SpawnProcess-30:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 43, in <module>
    @router.post("/create", summary="创建系统配置", dependencies=[DependSuperUser])
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 944, in decorator
    self.add_api_route(
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 883, in add_api_route
    route = route_class(
            ^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/routing.py", line 513, in __init__
    self.dependant = get_dependant(path=self.path_format, call=self.endpoint)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 268, in get_dependant
    sub_dependant = get_param_sub_dependant(
                    ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 111, in get_param_sub_dependant
    return get_sub_dependant(
           ^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 147, in get_sub_dependant
    sub_dependant = get_dependant(
                    ^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 250, in get_dependant
    endpoint_signature = get_typed_signature(call)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/fastapi/dependencies/utils.py", line 208, in get_typed_signature
    signature = inspect.signature(call)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/inspect.py", line 3263, in signature
    return Signature.from_callable(obj, follow_wrapped=follow_wrapped,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/inspect.py", line 3011, in from_callable
    return _signature_from_callable(obj, sigcls=cls,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/inspect.py", line 2456, in _signature_from_callable
    raise TypeError('{!r} is not a callable object'.format(obj))
TypeError: <ContextVar name='user_id' default=0 at 0x105847650> is not a callable object
2025-08-11 18:19:36 - WARNING - WatchFiles detected changes in 'app/api/v1/sys_config/sys_config.py'. Reloading...
Process SpawnProcess-31:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 46, in <module>
    user_id: int = Depends(CTX_USER_ID),
                           ^^^^^^^^^^^
NameError: name 'CTX_USER_ID' is not defined
2025-08-11 18:19:49 - WARNING - WatchFiles detected changes in 'app/api/v1/sys_config/sys_config.py'. Reloading...
Process SpawnProcess-32:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 83, in <module>
    user_id: int = Depends(CTX_USER_ID),
                           ^^^^^^^^^^^
NameError: name 'CTX_USER_ID' is not defined
2025-08-11 18:20:03 - WARNING - WatchFiles detected changes in 'app/api/v1/sys_config/sys_config.py'. Reloading...
Process SpawnProcess-33:
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/_subprocess.py", line 80, in subprocess_started
    target(sockets=sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 66, in run
    return asyncio.run(self.serve(sockets=sockets))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1518, in uvloop.loop.Loop.run_until_complete
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 70, in serve
    await self._serve(sockets)
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/server.py", line 77, in _serve
    config.load()
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/config.py", line 435, in load
    self.loaded_app = import_from_string(self.app)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/venv/lib/python3.11/site-packages/uvicorn/importer.py", line 19, in import_from_string
    module = importlib.import_module(module_str)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.13/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/__init__.py", line 7, in <module>
    from app.core.init_app import (
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/core/init_app.py", line 10, in <module>
    from app.api import api_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/__init__.py", line 3, in <module>
    from .v1 import v1_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/__init__.py", line 13, in <module>
    from .sys_config.sys_config import router as sys_config_router
  File "/Users/yang/PycharmProjects/vue-fastapi-admin/app/api/v1/sys_config/sys_config.py", line 83, in <module>
    user_id: int = Depends(CTX_USER_ID),
                           ^^^^^^^^^^^
NameError: name 'CTX_USER_ID' is not defined
2025-08-11 18:20:16 - WARNING - WatchFiles detected changes in 'app/api/v1/sys_config/sys_config.py'. Reloading...
2025-08-11 18:20:17 - INFO - Started server process [70584]
2025-08-11 18:20:17 - INFO - Waiting for application startup.
2025-08-11 18:20:17.307 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:20:17.307 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:20:17 - INFO - Application startup complete.
2025-08-11 18:20:17 - INFO - 127.0.0.1:52299 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:20:18 - INFO - 127.0.0.1:52369 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:20:29 - WARNING - WatchFiles detected changes in 'app/api/v1/sys_config/sys_config.py'. Reloading...
2025-08-11 18:20:29 - INFO - Shutting down
2025-08-11 18:20:29 - INFO - Waiting for application shutdown.
2025-08-11 18:20:29 - INFO - Application shutdown complete.
2025-08-11 18:20:29 - INFO - Finished server process [70584]
2025-08-11 18:20:30 - INFO - Started server process [70754]
2025-08-11 18:20:30 - INFO - Waiting for application startup.
2025-08-11 18:20:30.276 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:20:30.276 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:20:30 - INFO - Application startup complete.
2025-08-11 18:20:42 - WARNING - WatchFiles detected changes in 'migrations/models/7_20250811182042_add_sys_config_table.py'. Reloading...
2025-08-11 18:20:42 - INFO - Shutting down
2025-08-11 18:20:42 - INFO - Waiting for application shutdown.
2025-08-11 18:20:42 - INFO - Application shutdown complete.
2025-08-11 18:20:42 - INFO - Finished server process [70754]
2025-08-11 18:20:43 - INFO - Started server process [70846]
2025-08-11 18:20:43 - INFO - Waiting for application startup.
2025-08-11 18:20:43.085 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:20:43.085 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:20:43 - INFO - Application startup complete.
2025-08-11 18:21:29 - INFO - 127.0.0.1:52744 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 18:21:29 - INFO - 127.0.0.1:52745 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 18:21:29 - INFO - 127.0.0.1:52746 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 18:21:29 - INFO - 127.0.0.1:52763 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:22:01 - INFO - 127.0.0.1:52814 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 18:22:01 - INFO - 127.0.0.1:52815 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 18:22:01 - INFO - 127.0.0.1:52816 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 18:22:01 - INFO - 127.0.0.1:52825 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:22:11 - INFO - 127.0.0.1:52848 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 18:22:11 - INFO - 127.0.0.1:52850 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 18:22:11 - INFO - 127.0.0.1:52849 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 18:22:11 - INFO - 127.0.0.1:52859 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:22:29 - WARNING - WatchFiles detected changes in 'scripts/add_frontend_settings_menu.py'. Reloading...
2025-08-11 18:22:29 - INFO - Shutting down
2025-08-11 18:22:29 - INFO - Waiting for application shutdown.
2025-08-11 18:22:29 - INFO - Application shutdown complete.
2025-08-11 18:22:29 - INFO - Finished server process [70846]
2025-08-11 18:22:29 - INFO - Started server process [71772]
2025-08-11 18:22:29 - INFO - Waiting for application startup.
2025-08-11 18:22:29.908 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:22:29.909 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:22:30 - INFO - Application startup complete.
2025-08-11 18:22:55 - WARNING - WatchFiles detected changes in 'scripts/init_default_configs.py'. Reloading...
2025-08-11 18:22:55 - INFO - Shutting down
2025-08-11 18:22:55 - INFO - Waiting for application shutdown.
2025-08-11 18:22:55 - INFO - Application shutdown complete.
2025-08-11 18:22:55 - INFO - Finished server process [71772]
2025-08-11 18:22:56 - INFO - Started server process [72029]
2025-08-11 18:22:56 - INFO - Waiting for application startup.
2025-08-11 18:22:56.137 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:22:56.137 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:22:56 - INFO - Application startup complete.
2025-08-11 18:23:10 - WARNING - WatchFiles detected changes in 'scripts/add_frontend_settings_menu.py', 'scripts/init_default_configs.py'. Reloading...
2025-08-11 18:23:10 - INFO - Shutting down
2025-08-11 18:23:10 - INFO - Waiting for application shutdown.
2025-08-11 18:23:10 - INFO - Application shutdown complete.
2025-08-11 18:23:10 - INFO - Finished server process [72029]
2025-08-11 18:23:10 - INFO - Started server process [72123]
2025-08-11 18:23:10 - INFO - Waiting for application startup.
2025-08-11 18:23:10.759 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:23:10.759 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:23:11 - INFO - Application startup complete.
2025-08-11 18:23:24 - WARNING - WatchFiles detected changes in 'scripts/show_final_menu_structure.py'. Reloading...
2025-08-11 18:23:24 - INFO - Shutting down
2025-08-11 18:23:24 - INFO - Waiting for application shutdown.
2025-08-11 18:23:24 - INFO - Application shutdown complete.
2025-08-11 18:23:24 - INFO - Finished server process [72123]
2025-08-11 18:23:24 - INFO - Started server process [72286]
2025-08-11 18:23:24 - INFO - Waiting for application startup.
2025-08-11 18:23:24.869 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:23:24.870 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:23:25 - INFO - Application startup complete.
2025-08-11 18:23:38 - WARNING - WatchFiles detected changes in 'scripts/show_final_menu_structure.py'. Reloading...
2025-08-11 18:23:38 - INFO - Shutting down
2025-08-11 18:23:39 - INFO - Waiting for application shutdown.
2025-08-11 18:23:39 - INFO - Application shutdown complete.
2025-08-11 18:23:39 - INFO - Finished server process [72286]
2025-08-11 18:23:39 - INFO - Started server process [72412]
2025-08-11 18:23:39 - INFO - Waiting for application startup.
2025-08-11 18:23:39.598 | INFO     | app.core.init_app:init_db:200 - Tortoise ORM initialized successfully
2025-08-11 18:23:39.598 | INFO     | app.core.init_app:init_db:203 - Skipping database migration as fields already exist
2025-08-11 18:23:39 - INFO - Application startup complete.
2025-08-11 18:26:24 - INFO - 127.0.0.1:53121 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 18:26:25 - INFO - 127.0.0.1:53122 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 18:26:25 - INFO - 127.0.0.1:53123 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 18:26:25 - INFO - 127.0.0.1:53133 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:26:30 - INFO - 127.0.0.1:53145 - "GET /api/v1/sys_config/frontend HTTP/1.1" 200 OK
2025-08-11 18:26:42 - INFO - 127.0.0.1:53150 - "POST /api/v1/sys_config/frontend/update HTTP/1.1" 200 OK
2025-08-11 18:26:45 - INFO - 127.0.0.1:53153 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:26:46 - INFO - 127.0.0.1:53156 - "GET /api/v1/sys_config/frontend HTTP/1.1" 200 OK
2025-08-11 18:27:02 - INFO - 127.0.0.1:53166 - "GET /api/v1/announcement/list?page=1&page_size=10 HTTP/1.1" 200 OK
2025-08-11 18:27:02 - INFO - 127.0.0.1:53169 - "GET /api/v1/sys_config/frontend HTTP/1.1" 200 OK
2025-08-11 18:38:57 - INFO - 127.0.0.1:53512 - "GET /api/v1/base/userinfo HTTP/1.1" 200 OK
2025-08-11 18:38:57 - INFO - 127.0.0.1:53514 - "GET /api/v1/base/usermenu HTTP/1.1" 200 OK
2025-08-11 18:38:57 - INFO - 127.0.0.1:53515 - "GET /api/v1/base/userapi HTTP/1.1" 200 OK
2025-08-11 18:38:58 - INFO - 127.0.0.1:53519 - "GET /api/v1/sys_config/frontend HTTP/1.1" 200 OK
2025-08-11 18:42:36 - INFO - 127.0.0.1:53655 - "GET /docs HTTP/1.1" 200 OK
2025-08-11 18:42:36 - INFO - 127.0.0.1:53655 - "GET /openapi.json HTTP/1.1" 200 OK
